curriculum/challenges/english/blocks/workshop-quincys-job-tips/68ea7e7b916e8270483689b6.md

```diff
@@ -0,0 +1,74 @@
+---
+id: 68ea7e7b916e8270483689b6
+title: Step 7
+challengeType: 0
+dashedName: step-7
+---
+
+# --description--
+
+Below the `h2` element, you will add another quote by Quincy. This time, the quote won't be part of a larger paragraph. Instead, the whole paragraph will be a quote. In order to distinguish quoted text like this, you should use the block quotation element: `blockquote`. In the browser, you'll see that the text is slightly indented.
+
+Here is an example of a block quotation element with quoted text:
+
+```html
+<blockquote>
+  The first thing you should consider about education is this is an economic decision.
+</blockquote>
+```
+
+Now, inside the first section, add a block quotation element below the `h2` element with the text `Can you imagine what it would be like to be a successful developer? To have built software systems that people rely upon?`.
+
+# --hints--
+
+Inside the first section, you should have a `blockquote` element below the `h2` element.
+
+```js
+assert.exists(document.querySelector('section:first-of-type > h2 + blockquote'));
+```
+
+Your blockquote should have the text `Can you imagine what it would be like to be a successful developer? To have built software systems that people rely upon?`.
```

Comment:
```suggestion
Your `blockquote` element should have the text `Can you imagine what it would be like to be a successful developer? To have built software systems that people rely upon?`.
```